avl_tree/AvlTree.java

```diff
@@ -0,0 +1,306 @@
+import java.util.Scanner;
+import java.util.NoSuchElementException;
+import java.util.InputMismatchException;
+
+public class AvlTree {
+
+	public static void main(String[] args) {
+		AVL avl = new AVL();
+		Scanner inp = new Scanner(System.in);
+		while (true) {

```

Comment:
@Hsankesara There is no divine thing about this, I've already explained it in your previous PR, I can do it again though.
Hardcode inputs, i.e. add or delete nodes yourself rather than taking inputs from user, show working of all public methods which can be accessed by user. Show working of the exceptions thrown by any of the methods (if any).